lib/components/FAB/FAB.js

```diff
@@ -8,8 +8,14 @@ import { NatIcon } from '../NatIcon';
 export const FAB = ({
   color, onPress, size, theme, type, value, icon,
 }) => {
+  const iconSizes = {
```

Comment:
This immutable piece of code is run on every render of the component

There are two options here:
- Put this code outside the component declaration
- Use React.useMemo like the snippet below
```
const iconSizes = React.useMemo(() => ({
  default: 30,
  medium: 40, 
  large: 50
}), []);
```